adapters/handlers/rest/configure_api.go

```diff
@@ -32,12 +32,15 @@ import (
 	"google.golang.org/grpc"
 
 	"github.com/KimMachineGun/automemlimit/memlimit"
+	armonmetrics "github.com/armon/go-metrics"
```

Comment:
I see that `armon/go-metrics` redirect to hashicorp go metrics on github. Would it be possible to have the hashicorp URL as a dependency ?

Reply:
Ah interesting. Problem is the raft go package we use, still using `armo/go-metrics`.

https://github.com/hashicorp/raft/blob/main/go.mod#L8